src/Facebook/Entities/AccessToken.php

```diff
@@ -178,7 +177,7 @@ public static function getAccessTokenFromCode($code, $appId = null, $appSecret =
   {
     $params = array(
       'code' => $code,
-      'redirect_uri' => '',
```

Comment:
I'm :-1: for removing trailing commas from arrays. It doesn't cause any errors and is super convenient when adding new items. One of the nice features of PHP that JavaScript got wrong. :)